clients/admin-ui/src/features/user-management/UserForm.tsx

```diff
@@ -41,8 +41,8 @@ export type FormValues = typeof defaultInitialValues;
 
 const ValidationSchema = Yup.object().shape({
   username: Yup.string().required().label("Username"),
-  first_name: Yup.string().label("First name"),
-  last_name: Yup.string().label("Last name"),
+  first_name: Yup.string().required().label("First name"),
+  last_name: Yup.string().required().label("Last name"),
```

Comment:
These fields aren't actually required in the backend. Rachel had filed a ticket that said they were, but I think that was actually just a bad UX thing, where if the password didn't meet requirements, it would look like the form failed (and in rachel's case, she didn't have a first or last name). I would remove the `.required()` unless we want to make the backend require it

Reply:
just saw that @rsilvery called this out in particular in the ticket, so here's what I would push for:

* Should first and last name be required? They are not in the backend, and I don't think in general apps require it, so I would say no
* I imagine Rachel requested this at first because it _looked_ like they were required back when password validation wasn't working. But now that password validation is fixed, you should be able to input a user without a first or last name just fine, and so I'd say that we don't need to make these required fields on the frontend

Reply:
@rsilvery - mind confirming that first/ last name shouldn't be a requirement on FE?

Reply:
I don't think it needs to be so long as the BE isn't requiring it.